packages/typespec-test/test/openai_generic/generated/typespec-ts/src/rest/models.ts

```diff
@@ -186,7 +186,7 @@ export interface ChatCompletionRequestMessage {
    */
   name?: string;
   /** The name and arguments of a function that should be called, as generated by the model. */
-  function_call?: object;
```

Comment:
this is because we have logic to fall back to schema.type and then it would be `object`. Generally this only works for fixed enum schema or some constants schema and its type would be relevant values.